README.md

```diff
@@ -103,48 +103,58 @@ To connect to AWS Keyspaces, the following environment variables need to be set:
 - `AWS_KEYSPACE` - Your AWS Keyspace name.
 - `CASSANDRA_HOST` - Cassandra host (e.g. cassandra.us-west-2.amazonaws.com).
 - `CASSANDRA_PORT` - Cassandra port (e.g. 9142).
-- `CASSANDRA_USERNAME` - Cassandra service user.
-- `CASSANDRA_PASSWORD` - Cassandra service password.
+- `AWS_ACCESS_KEY_ID` - Your AWS Access Key ID.
+- `AWS_SECRET_ACCESS_KEY` - Your AWS Secret Access Key.
+- `AWS_DEFAULT_REGION` - Your AWS Default region. (e.g. us-west-2, it is needed for `stateless-verification-tool`)
 - `SSL_CERTFILE` - The path to your SSL certificate for AWS Keyspaces.
 
 > 🗒️ **Note 1:** For convenience, an SSL certificate is provided in this repository and can be found at [/uptime_service_validation/database/aws_keyspaces/cert/sf-class2-root.crt](/uptime_service_validation/database/aws_keyspaces/cert/sf-class2-root.crt). Alternatively, the certificate can also be downloaded directly from AWS. Detailed instructions for obtaining the certificate are available in the AWS Keyspaces documentation, which you can access [here](https://docs.aws.amazon.com/keyspaces/latest/devguide/using_python_driver.html#using_python_driver.BeforeYouBegin).
 
-> 🗒️ **Note 2:** Docker image already includes cert and has `AWS_SSL_CERTIFICATE_PATH` set up, however it can be overriden by providing this env variable to docker.
+> 🗒️ **Note 2:** Docker image already includes cert and has `SSL_CERTFILE` set up, however it can be overriden by providing this env variable to docker.
```

Comment:
Such notes talking about software placed in different repo bear an extremely high risk of becoming unexpectedly outdated at some point in the future. I'd rather find here a link to the documentation of the docker image explaining this in detail.

Reply:
Ah, okay.